pytorch_lightning/loggers/base.py

```diff
@@ -195,10 +195,7 @@ def agg_and_log_metrics(self, metrics: Dict[str, float], step: Optional[int] = N
             metrics: Dictionary with metric names as keys and measured quantities as values
             step: Step number at which the metrics should be recorded
         """
-        agg_step, metrics_to_log = self._aggregate_metrics(metrics=metrics, step=step)
```

Comment:
If we don't call `self._aggregate_metrics` anymore, shouldn't we delete the method too?

Reply:
Yup, was gonna do in a second PR but just added to this one.